src/opera/pge/disp_s1/templates/disp_s1_measured_parameters.yaml

```diff
@@ -166,3 +278,119 @@ metadata/dolphin_software_version:
   attribute_type: processingInformation
   description: Version of the Dolphin software used to generate the product
   display_name: DolphinSoftwareVersion
+
+metadata/pge_runconfig:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: The full PGE runconfig YAML file used to generate the product.
+  display_name: PGERunConfig
+metadata/algorithm_parameters_yaml:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: The full algorithm parameters YAML file used to generate the product.
+  display_name: AlgorithmParametersYAML
+metadata/dolphin_workflow_config:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: The configuration parameters used by `dolphin` during the processing.
+  display_name: DolphinWorkflowConfig
+
+
+# TODO: These are new in SAS v0.5.3 - Descriptions sourced from code, not spec. attribute_data_type fields are guessed.
+#  Need to double check
+
+metadata/algorithm_theoretical_basis_document_id:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: Document identifier for Algorithm Theoretical Basis Document (ATBD).
+  display_name: AlgorithmTheoreticalBasisDocumentID
+metadata/ceos_atmospheric_phase_correction:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: Method used to correct for atmosphere phase noise.
+  display_name: CEOSAtmosphericPhaseCorrection
+metadata/ceos_gridding_convention:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: Whether a consistent gridding/sampling frame is used for ascending/descending frames.
+  display_name: CEOSGriddingConvention
+metadata/ceos_insar_pair_baseline_criteria_information:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: InSAR pair baseline selection criteria.
+  display_name: CEOSInSARPairBaselineCriteriaInformation
+metadata/ceos_ionospheric_phase_correction:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: Method used to correct for ionospheric phase noise.
+  display_name: CEOSIonosphericPhaseCorrection
+metadata/ceos_noise_removal:
+  attribute_data_type: string
+  attribute_type: processingInformation
+  description: Flag if noise removal* has been applied (Y/N). Metadata should include the noise removal algorithm and reference to the algorithm as URL or DOI.
+  display_name: CEOSNoiseRemoval
+metadata/ceos_persistent_scatterer_amplitude_dispersion_threshold:
+  attribute_data_type: float
+  attribute_type: processingInformation
+  description: "TBA"  #TODO: ask Scott S - this is a duplicate description in code
```

Comment:
Instead of "TBA" as a placeholder here, lets just reiterate the name of the variable: `description: CEOS persistent scatterer amplitude dispersion threshold`

Reply:
that is also good. https://github.com/opera-adt/disp-s1/pull/252/files would be the way I'd correct it, but these changes are no better than your reiteration